src/main/java/org/itsallcode/matcher/MultilineTextMatcher.java

```diff
@@ -83,13 +84,13 @@ protected void describeMismatchSafely(final String text, final Description misma
 				mismatchDescription.appendText(">>> ");
 			}
 			mismatchDescription.appendText(line);
-			mismatchDescription.appendText(System.lineSeparator());
+			mismatchDescription.appendText(LINE_SEPARATOR);
```

Comment:
Since this is debug output it would be safer to fix the test. This could lead to garbled output on non-Unix platforms.

Reply:
Fixed.